packages/rum/src/types/sessionReplayUtils.ts

```diff
@@ -0,0 +1,84 @@
+import type * as SessionReplay from './sessionReplay'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏this file could be renamed `sessionReplayConstants` instead of `Utils`